jena-arq/src/main/java/org/apache/jena/riot/RDFWriterRegistry.java

```diff
@@ -301,22 +301,32 @@ public static boolean contains(RDFFormat format)
 
     /** All registered graph formats */
     public static Collection<RDFFormat> registeredGraphFormats() {
-        return Collections.unmodifiableSet(registryGraph.keySet()) ;
+        return Set.copyOf(registryGraph.keySet()) ;
     }
 
     /** All registered dataset formats */
     public static Collection<RDFFormat> registeredDatasetFormats() {
-        return Collections.unmodifiableSet(registryDataset.keySet()) ;
+        return Set.copyOf(registryDataset.keySet()) ;
+    }
+
+    /** All registered formats */
+    public static Collection<RDFFormat> registeredFormats() {
+        return registered();
     }
 
     /** All registered formats */
     public static Collection<RDFFormat> registered() {
         Set<RDFFormat> x = new HashSet<>() ;
         x.addAll(registryGraph.keySet()) ;
         x.addAll(registryDataset.keySet()) ;
-        return Collections.unmodifiableSet(x) ;
+        return Set.copyOf(x) ;
     }
```

Comment:
Let's deprecate this and remove it in a later release.

`registeredFormats` is the implementation.
`registered()` calls `registeredFormats`.

Reply:
Done